classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -106,6 +106,8 @@ public function post( $request ) {
 				$label_meta[ 'package_name' ] = __( 'Unknown package', 'woocommerce-services' );
 			}
 
+			$label_meta['is_letter'] = $package['is_letter'];
```

Comment:
Two small changes here. One is just stylistic with spaces between bracketed strings and the other is to just assign a default value to avoid any undefined warnings etc. I went with null here based on our discussion, but don't think false would be bad here as it would be functionally the same and is nicer to be able to know it's always a boolean. This update should require the tests to be updated from undefined to null (or false if you wanted to go that route).

```suggestion
			$label_meta[ 'is_letter' ] = isset( $package[ 'is_letter' ] ) ? $package[ 'is_letter' ] : null;
```

Reply:
Done! I used `false` as default and updated the test with a mix of `true`, `false` and `undefined`.